presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -924,6 +924,22 @@ public void testSelectWithOffset()
                         Optional.of(new Offset("2")),
                         Optional.empty()));
 
+        assertStatement("SELECT * FROM table1 OFFSET 2",
```

Comment:
This change should not belong to this commit since it's the syntax and parser change. It should be moved the first commit `Add support for OFFSET syntax in grammar and AST`. So the first commit shall contain #723 and #746.

Reply:
will do, thx